Modix.Data/Models/BehaviourConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Modix.Data.Models
+{
+    public class BehaviourConfiguration
```

Comment:
Can we just call this `Configuration` or `AppConfiguration` or `ConfigurationSetting` or similar.

`Behavior` has a specific connotation in `Modix.Services` within #91, in the form of `IBehavior` and `BehaviorBase`.

I think `ConfigurationSetting` is my favorite.

Reply:
It's a configuration specific for the bot's behaviour, if there are other configurations that fall outside of that I'd rather we reserve that name for them

Reply:
My point is if we have an `IBehavior` interface, and `BehaviorConfiguration` services and such, it's implied that those are directly related, when really they won't be. If you've got another idea for what to call an `IBehavior`, we could change that instead.